astroid/arguments.py

```diff
@@ -134,14 +135,7 @@ def _unpack_args(self, args, context: InferenceContext | None = None):
         context.extra_context = self.argument_context_map
         for arg in args:
             if isinstance(arg, nodes.Starred):
-                try:
-                    inferred = next(arg.value.infer(context=context))
-                except InferenceError:
-                    values.append(Uninferable)
-                    continue
-                except StopIteration:
-                    continue
-
+                inferred = safe_infer(arg.value, context=context)
```

Comment:
Isn't there the same flaw for unpacking `**kwargs` in `_unpack_keywords` just above this?

Reply:
I know we have to draw the line somewhere, but it's essentially the same test case :-)

Reply:
Alright, I'll go ahead and fix that too :D